### PR TITLE
Apr 906 pat only delivering frameworks

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json [hunk 1]
```diff
@@ -1,590 +1,846 @@
...
+                {
+                    "QuestionId": "PAT-23",
+                    "QuestionTag": "PAT-Type-Of-Training-supporting",
+                    "Label": "What type of apprenticeship training will your organisation offer?",
+                    "ShortLabel": "",
+                    "QuestionBodyText": "<p class='govuk-body'>Select all that apply.</p>",
+                    "Hint": "",
+                    "Input": {
+                        "Type": "CheckboxList",
+                        "Options": [
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json]: Looks like repeated values here in the options - are these being filtered elsewhere?

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json [hunk 2]
```diff
@@ -1,590 +1,846 @@
...
+                            "QuestionTag": null,
+                            "Contains": "Part apprenticeship standards"
+                        }
+                    ],
+                    "ConditionMet": false
+                },
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6060",
+                    "Conditions": [
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json]: As above - how will this condition be triggered if there's one above that goes to page 6200?

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json]: I can't see any repeated values.  The words are different...

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json]: the one above has different words in the contains

**Pull request closed without merging**